testing/constraints-3.9.txt

```diff
@@ -3,5 +3,4 @@
 httplib2==0.19.0
 google-auth
 google-auth-httplib2==0.2.0
-google-api-core==2.15.0
```

Comment:
For prototyping purposes only, revert once https://github.com/googleapis/python-api-core/pull/638 is merged and released to PyPI